### DISCUSSION TITLE
اضافه شدن ورود با کد یکبار مصرف

### DISCUSSION THREAD

**Discussion by mohammad-ashrafi2020** [mohammad-ashrafi2020/DigiLearn#2] [Ideas]
به پروژه این قابلیت اضافه بشه که 2 روش لاگین در دسترس باشه یکی با شماره تلفن و رمز ثابت و یکی هم با کد یکبار مصرف  که به شماره کاربر ارسال میشه

**Comment by mohammad-ashrafi2020** [comment]
برای سرویس اش هم میتونیم از کاوه نگار استفاده کنیم که rest api داره و پیاده سازیش نصب به بقه provider ها ساده تره

**Reply by Sajjad3223** [reply]
سلام استاد، باید از OTP کاوه نگار استفاده کنیم؟ یا از سرویس ارسال پیامک
و اینکه استاد provider ارزون تری سراغ ندارین، کاوه نگار بالاس هزینه هاش

**Reply by mohammad-ashrafi2020** [reply]
سلام، سرویس هایه دیگه ای مثل ملی پیامک هم هست که احراز حویت اش یه کم طول میکشه در عوض هزینه هاش کم تره نسبت به بقیه ، ما برای سایت همون اکثرا از ملی پیامک  استفاده میکنیم

**Reply by Sajjad3223** [reply]
ممنون استاد
